Automattic/wp-calypso
# Adjust cancellation offer flow for Akismet branding

## Proposed Changes

As we introduce Akismet cancellation offers - we want to adjust the visual branding in the flow. With this PR, we make sure we distinguish Akismet and Jetpack in text and assets used in the flow:

(1)
![CleanShot 2023-10-10 at 12 29 45@2x](https://github.com/Automattic/wp-calypso/assets/8419292/36bf2601-b563-48a4-9178-cadf8066e119)

(2)
![CleanShot 2023-10-10 at 12 29 20@2x](https://github.com/Automattic/wp-calypso/assets/8419292/aa98609f-9e9f-4c9a-b375-5c7fd6a5a51e)

(3)
![CleanShot 2023-10-10 at 12 19 25@2x](https://github.com/Automattic/wp-calypso/assets/8419292/bd2f6bf4-9045-4aed-82b8-14cd2cef72de)


## Testing Instructions

* Follow the steps described here: D124824-code
* Purchase Akismet Pro 10K (montly or yearly)
* Run Calypso Live link (or fetch locally) and go to `/me/purchases`
* Try to go through the cancellation flow and make sure it matches with screenshots above

## Pre-merge Checklist
